scripts/aws/alarms.yaml

```diff
@@ -359,4 +363,8 @@ Resources:
             - !GetAtt ClusterStatusYellow.Arn
             - !GetAtt FreeStorageSpace.Arn
             - !If [ HasMasterNode, !GetAtt MasterJVMMemoryPressure.Arn, '']
-            - !If [ HasMasterNode, !GetAtt MasterCPUUtilization.Arn, '']            
+            - !If [ HasMasterNode, !GetAtt MasterCPUUtilization.Arn, '']
+        LogGroupsNames: !Join
+          - ','
+          - - !Ref LogGroup
```

Comment:
@FeliceMente Rinomina in LogGroupName

Reply:
LogGroupsNames è il nome che viene cercato dalla transform function, non funzionerebbe se fosse diverso